release.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
_Unrelated to line number_

Maybe rename it to `prepare_release.sh` since this is not releasing anything, just preparing it for a release